### PR TITLE
fix: add new icons for search and networks nav item

### PR DESCRIPTION
**Pull request by two24studios**
# Summary

Added:
- network (for Networks in main nav)
- profile-team (for developer teams in KSearch)

Updated:
- stack icon (for api product versions in KSearch)

### CODE REVIEW

File: svg/solid/stack.svg
Comment (jillztom) [inline svg/solid/stack.svg:1]: We need to make sure we are not using it in the "Copy" button components

Comment (two24studios) [reply svg/solid/stack.svg:1]: This icon replaces one that was even more similar to `Copy` so hopefully this will be better.

Comment (adamdehaven) [reply svg/solid/stack.svg:1]: @jillztom Jason essentially wants to replace the existing Copy Icon - the new one will take up the same dimensions (controlled by the component) but will change the look of the icon obviously as it gets picked up across repos.

You good with this?

Comment (jillztom) [reply svg/solid/stack.svg:1]: As long as it's a desired outcome, I'm good

Comment (adamdehaven) [reply svg/solid/stack.svg:1]: I confused myself -- the copy icon isn't changing in this PR

image

**Review by adamdehaven — APPROVED** [review]
lgtm

**Pull request merged**